build.psm1

```diff
@@ -2117,8 +2127,10 @@ function Start-CrossGen {
     $crossGenRuntime = if ($Environment.IsWindows) {
         if ($Runtime -match "-x86") {
             "win-x86"
-        } else {
+        } elseif ($Runtime -match "-x64") {
             "win-x64"
+        } elseif (!$env:PROCESSOR_ARCHITECTURE -match "arm") {
```

Comment:
> !$env:PROCESSOR_ARCHITECTURE -match "arm"

`!$env:PROCESSOR_ARCHITECTURE` will be evaluated first in this case.

Reply:
Will fix